website/package.json

```diff
@@ -42,26 +39,27 @@
     "@docusaurus/theme-mermaid": "3.4.0",
     "@mdx-js/react": "3.0.1",
     "clsx": "1.2.1",
-    "prism-react-renderer": "2.3.1",
+    "prism-react-renderer": "2.4.0",
     "react": "18.3.1",
     "react-dom": "18.3.1",
-    "rehype-katex": "7.0.0"
+    "rehype-katex": "7.0.1"
   },
   "devDependencies": {
-    "@docusaurus/types": "3.4.0",
+    "@docusaurus/types": "3.5.2",
     "prettier": "3.3.3",
     "remark-cli": "12.0.1",
     "remark-inline-links": "7.0.0",
-    "remark-lint-heading-capitalization": "1.1.0",
+    "remark-lint-heading-capitalization": "1.2.0",
     "remark-lint-match-punctuation": "0.2.1",
     "remark-lint-no-url-trailing-slash": "3.0.1",
     "remark-math": "6.0.0",
     "remark-preset-lint-consistent": "6.0.0",
     "remark-preset-lint-markdown-style-guide": "6.0.0",
     "remark-preset-lint-recommended": "7.0.0",
+    "remark-remove-url-trailing-slash": "2.0.0",
     "remark-sort-definitions": "2.0.0",
     "remark-toc": "9.0.0",
     "remark-wiki-link": "2.0.1"
   },
-  "packageManager": "bun@1.1.21"
+  "packageManager": "bun@1.1.29"
```

Comment:
https://github.com/renovatebot/renovate/issues/29941 seems like there should be ways to force this? this setting was not respected AFAICT